projects/gui/src/newtournamentdialog.cpp

```diff
@@ -235,7 +235,11 @@ Tournament* NewTournamentDialog::createTournament(GameManager* gameManager) cons
 	t->setName(ui->m_nameEdit->text());
 	t->setSite(ui->m_siteEdit->text());
 	t->setVariant(ui->m_gameSettings->chessVariant());
-	t->setPgnOutput(ui->m_pgnoutEdit->text());
+	QString pgnName = ui->m_pgnoutEdit->text();
```

Comment:
I think this is the wrong place to read the default PGN file path. Instead, `NewTournamentDialog::readSettings()` should have `ui->m_pgnoutEdit->setText(QSettings().value("pgn/default_tournament_pgn_file").toString());` so that the user could see and change the PGN output file before starting the tournament.